modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java

```diff
@@ -240,6 +240,12 @@ public class CacheMetricsImpl implements CacheMetrics {
     /** Conflict resolver merged entries count. */
     private LongAdderMetric rslvrMergedCnt;
 
+    /** PutAllConflict time. */
+    private final HistogramMetricImpl putAllConflictTime;
```

Comment:
Could you please move code of new metrics closer to similar metrics? I mean, this fields definition, metric creation, metrics reset, "add*" methods, etc.